pandas/core/groupby/groupby.py

```diff
@@ -3511,11 +3515,12 @@ def resample(self, rule, *args, include_groups: bool = True, **kwargs) -> Resamp
 
         Returns
         -------
-        pandas.api.typing.DatetimeIndexResamplerGroupby,
-        pandas.api.typing.PeriodIndexResamplerGroupby, or
-        pandas.api.typing.TimedeltaIndexResamplerGroupby
+        pandas.api.typing.DatetimeIndexResampler
             Return a new groupby object, with type depending on the data
-            being resampled.
+            being resampled:
+            - pandas.api.typing.DatetimeIndexResamplerGroupby
+            - pandas.api.typing.PeriodIndexResamplerGroupby
+            - pandas.api.typing.TimedeltaIndexResamplerGroupby
```

Comment:
This doesn't seem to be rendering correctly: https://pandas.pydata.org/preview/pandas-dev/pandas/57862/docs/reference/api/pandas.core.groupby.DataFrameGroupBy.resample.html#pandas.core.groupby.DataFrameGroupBy.resample

Also, the type is incorrect now. Why don't you leave as type simply: `DatetimeIndexResampler, PeriodIndexResampler or TimdeltaResampler`

And in the description you can have something like `Resampler object for the type of the index`.

Reply:
Updated. But i'll hold this up and wait for the new flow regarding alll this doc-related thingy